### PR TITLE
Update docs to align with major warp packages

### PR DESCRIPTION
**Pull request by magnuh**
- Update to major warp pkgs
- Rename the Vue Input component to Text Field 
…and fixed misc other oddities and typos related to Text Field and Text Area.

### CODE REVIEW

**Review by BalbinaK — COMMENTED** [review]
Great stuff! Could you make a note that this PR should not be merged until we update the tech-docs with 1.0.0 versions? Here we are still pointing at the old alpha versions that are compatible with Finn prod tokens, so having these instructions + new component API might be confusing.

File: package.json [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
   },
   "devDependencies": {
-    "@warp-ds/component-classes": "1.0.0-alpha.116",
-    "@warp-ds/elements": "1.0.0-alpha.27",
-    "@warp-ds/uno": "1.0.0-alpha.49",
-    "@warp-ds/vue": "^1.0.0-alpha.43",
+    "@warp-ds/css": "^1.0.0",
+    "@warp-ds/elements": "1.0.0",
+    "@warp-ds/uno": "1.0.0",
+    "@warp-ds/vue": "^1.0.0",
```
Comment (siljec) [inline package.json]: Should we have `^` for all warp packages? To make sure we get newest changes up to a new major version? 🤔

Comment (BalbinaK) [reply package.json]: Good catch! Fixed in [9e6f645](https://github.com/warp-ds/tech-docs/pull/96/commits/9e6f64599adda2460766a8cb11d8f0880856d3a9)

**Review by siljec — APPROVED** [review]
Looks good 🔥 🙌  I just had a question

**Comment by BalbinaK** [comment]
> Looks good 🔥 🙌 I just had a question

The one that got fixed in [9e6f645](https://github.com/warp-ds/tech-docs/pull/96/commits/9e6f64599adda2460766a8cb11d8f0880856d3a9)? 😄

**Comment by siljec** [comment]
> > Looks good 🔥 🙌 I just had a question
> 
> The one that got fixed in [9e6f645](https://github.com/warp-ds/tech-docs/pull/96/commits/9e6f64599adda2460766a8cb11d8f0880856d3a9)? 😄

Yes, exactly, you were to qucik so I didn't catch that it was fixed already 😅 @BalbinaK

**Pull request merged**